protocol/contracts/Utils.sol

```diff
@@ -0,0 +1,19 @@
+
+pragma solidity ^0.5.17;
+pragma experimental ABIEncoderV2;
+
+import "@openzeppelin/contracts/math/SafeMath.sol";
+
+library Utils {
```

Comment:
From what I can see, the Chainlink Oracles are returning 8 decimals usually.  As decimal remapping is happening in multiple places, and could need to happen in future pools, I thought it a good idea to introduce this helper. 

The idea is that you can feed in a uint256 price, the decimals it is stored in, and the target decimals you would like to reach.

Assuming a target of 4, and a decimals of 2, we should see 100-00 turn in to 100-0000
Assuming a target of 2, and a decimals of 4, we should see 100-0000 turn in to 100-00
Assuming a target of 4, and a decimals of 4, we should see 100-0000 turn in to 100-0000

Reply:
We will obviously lose accuracy if we ever add an asset with >18 decimals, but so far I have not seen one of those.